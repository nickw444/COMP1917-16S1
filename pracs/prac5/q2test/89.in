89
5 -6 8 3 -2 7 4 -2 1 -1 -9 4 -9 8 -5 7 2 4 5 1 -1 9 -4 -2 -4 -10 7 -8 -4 8 -6 -5 4 2 2 9 -10 7 1 8 -7 -1 -4 -5 -1 -3 1 -10 10 -3 7 8 2 -3 5 -2 -3 -10 9 -10 -1 -8 2 -6 -1 -5 10 7 -4 6 6 1 -2 4 -3 -5 -6 7 -10 5 -7 4 -3 -6 4 -9 1 4 8
