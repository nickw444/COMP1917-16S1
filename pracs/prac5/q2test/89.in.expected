average of array: 0.011236
